run.py

```diff
@@ -1,17 +1,20 @@
 from data_handler import DataHandler
 from argparse import ArgumentParser
+from data_outputter import get_outputter
 
 arg_parser = ArgumentParser()
 arg_parser.add_argument('--sample', '-s', nargs='?', metavar='SIZE', type=int, default=0,
                         help='sample size, default: 0 (full run)')
 arg_parser.add_argument('--no-scrape', '-ns', action='store_true',
                         help='skip scraping during run')
-
+arg_parser.add_argument('--push', '-p', action='store_true', 
+                        help='push output to rust server')
 args = arg_parser.parse_args()
 
 if __name__ == '__main__':
     dh = DataHandler(
         sample=args.sample,
-        no_scrape=args.no_scrape
+        no_scrape=args.no_scrape,
+        outputter=get_outputter(args.push)
```

Comment:
I'd prefer just passing `args.push` to DataHandler and setting the outputter inside `__init__()`

Reply:
Jeg kan godt ændrer det da vi ikke forventer flere output varianter. Men havde valgt dette design for at kunne inject hvilken som helst strategi.